pom.xml

```diff
@@ -541,6 +540,19 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
+            <repositories>
+                <repository>
+                    <id>apache.snapshots</id>
+                    <name>Apache Development Snapshot Repository</name>
+                    <url>https://repository.apache.org/content/repositories/orgapachecalcite-1078</url>
```

Comment:
This is the repository for 1.22.0rc1
They will release 1.22.0 officially by the end of this week